src/@sdk/react/types.ts

```diff
@@ -18,7 +18,7 @@ export type Options<T extends keyof SaleorAPI> = SaleorAPI[T] extends (
   _: any,
   options: infer V
 ) => any
-  ? V
+  ? V & { skip?: boolean }
```

Comment:
I'd move this to query factory typings (ie options: Options<T> & HookOptions), this is more of a helper method and it shouldn't have hardcoded types here.